⭐️ action to scan GitHub organization and repositories (#30)
mondoohq/actions: github-repo/README.md

# Mondoo GitHub Repository Action

> A GitHub Action for using Mondoo to scan Kubernetes clusters for security vulnerabilities and misconfigurations. This Action can be used to audit individual GitHub repositories.

Reviewer: ```suggestion
A GitHub Action for using Mondoo to scan GitHub repositories for security misconfigurations such as branch protection, CI tests, required code-review, and more. This Action can be used to audit individual GitHub repositories.
```

## Properties

> The Kubernetes Cluster Action has properties which are passed to the underlying image. These are passed to the action using `with`.

Reviewer: ```suggestion
The GitHub repository Action has properties that are passed to the action using `with`.
```
